styles/components/_scoreTable.scss

```diff
@@ -0,0 +1,36 @@
+.scores {
+  background-color: $color-primary;
+  border-radius: $border-radius;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
This value should be declared in _variables.scss, cause its implemented in many files